paddle/fluid/framework/new_executor/interpretercore.cc

```diff
@@ -998,6 +998,7 @@ void InterpreterCore::Prepare(
     const std::vector<std::string>& feed_names,
     const std::vector<framework::LoDTensor>& feed_tensors,
     bool prepare_feed) {
+  VLOG(1) << "################# InterpreterCore::Prepare";
```

Comment:
if needless, recommend to remove debug info, or use higher log level

Reply:
done